easybuild/easyconfigs/m/magma/make.inc

```diff
@@ -0,0 +1,52 @@
+#//////////////////////////////////////////////////////////////////////////////
+#   -- MAGMA (version 1.6.1) --
+#      Univ. of Tennessee, Knoxville
+#      Univ. of California, Berkeley
+#      Univ. of Colorado, Denver
+#      @date January 2015
+#
+#  make.inc updated by @twrobinson and @gppezzi for compilation on XC30 (Piz Daint)
+#
+#//////////////////////////////////////////////////////////////////////////////
+
+
+# GPU_TARGET contains one or more of Tesla, Fermi, or Kepler,
+# to specify for which GPUs you want to compile MAGMA:
+#     Tesla  - NVIDIA compute capability 1.x cards (no longer supported in CUDA 6.5)
+#     Fermi  - NVIDIA compute capability 2.x cards
+#     Kepler - NVIDIA compute capability 3.x cards
+# The default is "Fermi Kepler".
+# See http://developer.nvidia.com/cuda-gpus
+#
+GPU_TARGET = Fermi Kepler
+
+CXX      := $(CXX)
+CC       := $(CC)
+NVCC     := nvcc
+FORT     := $(F90)
+
+ARCH      = ar
+ARCHFLAGS = cr
+RANLIB    = ranlib
+
+# Use -fPIC to make shared (.so) and static (.a) library;
+# can be commented out if making only static library.
+FPIC      = -fPIC
+
+CFLAGS    += -DADD_ -DMAGMA_SETAFFINITY
```

Comment:
Magma provides some examples of make.inc, we basically took the one for openblas (and removed openblas). MAGMA_SETAFFINITY comes from that example and is present on most of the other make.inc they provide. Our changes are really minor, they are mostly small things in order to be compatible with the Cray wrappers + EB toolchain (e.g. removed the -W* from CFLAGS, added missing CXXFLAGS variable). You can check the original file here https://gist.github.com/gppezzi/4f60e4c335bd74559b8c